schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int NOT NULL PRIMARY KEY,
+    name varchar(100),
+    date_of_birth date,
+    escape_attempts int,
+    neutred boolean,
+    weight_kg float
 );
```

Comment:
- [ ] As required for the project, kindly ensure that the `weight_kg` from your `animals` table has the correct data type as it is supposed to be of type `decimal` but `numeric` data type is also accepted. Make sure to fix it